### DISCUSSION TITLE
Can't access my endpoint, it times out

### DISCUSSION THREAD

**Discussion by EfosaSO** [supabase/supabase#2608] [Questions] [answered]
Been having issues trying to access my projects endpoint. I've also tried restarting the server but no luck.

![image](https://user-images.githubusercontent.com/22990506/127308920-8939971c-7d9b-4857-981f-36223f02ba40.png)

**Comment by molimat** [comment]
I'm experiencing the same problem on some of my projects, not all of them.

**Reply by ltanady** [reply]
@molimat it's fixed now can you try it?

**Reply by molimat** [reply]
Still having same problem with my server on sa-east-1, getting timeout or CORS problem. The Table Editor doesn't load as well, pops an API connection error.

**Reply by EfosaSO** [reply]
It's fixed for me now

**Reply by rphly** [reply]
Just fixed for me now

**Reply by yamin8000** [reply]
I'm still not sure if it's a problem from supabase side or my ISP blocked it because if I use a VPN based in England I can access my data but from my actual IP from Iran I can't access.

**Comment by rphly** [comment]
I've noticed this problem also a few days ago. Wondering if the Supabase team could maybe let us know what's going on and if there's anything we can do to mitigate the outages on our end (if at all), are we doing something wrong?

**Comment by inian** [comment, marked as answer]
This was a temporary incident - https://status.supabase.io/incidents/gn8vshqh65kz

If you are still facing an issue with your project, reach out to beta@supabase.io with your project details
